### ISSUE TITLE
代码疑问：这里保存模型参数是origin_head_weight，为啥不是default_head_weight吗？保存origin_head_weight后的话，也没有到重新加载。

### ISSUE THREAD

**Issue by sunlei198911** [hiyouga/ChatGLM-Efficient-Tuning#255]
def replace_model(model: AutoModelForCausalLMWithValueHead, target: Literal["default", "reward"]) -> None:
    if target == "reward": # save original head temporarily
        valuehead_state_dict = model.v_head.state_dict()

        setattr(model, "origin_head_weight", valuehead_state_dict["summary.weight"])
        setattr(model, "origin_head_bias", valuehead_state_dict["summary.bias"])

        #model.pretrained_model.set_adapter(target) # set the LoRA adapter to be active
         model.v_head.load_state_dict({
        "summary.weight": getattr(model, "{}_head_weight".format(target)),
        "summary.bias": getattr(model, "{}_head_bias".format(target))
    })

**Comment by hiyouga** [comment]
感谢指正，是我疏忽了，目前已经更新
